test/integration/server/dal/connection-manager.ts

```diff
@@ -1,5 +1,5 @@
 import { ConnectionOptions, createPolarisConnection } from '@enigmatis/polaris-typeorm';
-import { polarisGraphQLLogger } from './logger';
+import { polarisGraphQLLogger } from '../utils/logger';
```

Comment:
Does a logger accounts as a util anyways?

Reply:
Not really, should we create a specific folder for it named "logger"?

Reply:
I'm not sure... Well lets leave it like this in the meanwhile.